### PR TITLE
Enable history and command editing in interactive mode in tn-cli.

### PR DESCRIPTION
**Pull request by aforge**

### CODE REVIEW

File: tn-cli/tn-cli.py [hunk 1]
```diff
@@ -24,6 +24,7 @@
     import queue
 import random
 import re
+import readline
```
Comment (or-else) [inline tn-cli/tn-cli.py]: Please add readline to requirements.txt.

**Comment by or-else** [comment]
It looks like readline is not available on Windows: https://bugs.python.org/issue26258

File: tn-cli/tn-cli.py (hunk 1 above)
Comment (or-else) [reply tn-cli/tn-cli.py]: There is something called pyreadline which exists on windows but seems to be abandoned:
https://pypi.org/project/pyreadline/

Comment (aforge) [reply tn-cli/tn-cli.py]: Good catch! Yes, I'll check pyreadline - we need to see if it's compatible with raw_input()/input().

Comment (or-else) [reply tn-cli/tn-cli.py]: Please also look at https://github.com/prompt-toolkit/python-prompt-toolkit
It seems to be well-supported albeit huge.

**Comment by aforge** [comment]
Incorporated prompt toolkit into this PR.

**Comment by or-else** [comment]
It it ready for review?

**Comment by aforge** [comment]
Correct. Please, take a look.

**Comment by or-else** [comment]
`python-prompt-toolkit` works fine but `tn-cli` in general is broken. Not because of this change, it's broken in devel as well.

**Pull request merged**